python/ml4ir/applications/ranking/tests/test_ml4ir_sanity.py

```diff
@@ -0,0 +1,266 @@
+import unittest
```

Comment:
@mohazahran Please add [numpy style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html) for all the methods and. as @jakemannix mentioned, [type hints](https://docs.python.org/3/library/typing.html).